client/my-sites/plans/current-plan/my-plan-card/index.jsx

```diff
@@ -29,7 +29,7 @@ const MyPlanCard = ( { action, isError, isPlaceholder, details, plan, tagLine, t
 				<div className="my-plan-card__icon">{ plan && <PlanIcon plan={ plan } /> }</div>
```

Comment:
Note that I get this when I have a backup product:

![](https://cldup.com/sYn0oJpMr2.png)

I guess it's expected before we land the ProductIcon component and use it instead?

Reply:
Well, both yes and no. I'm going to create another 'integration' PR once it's all merged so that we're using the `ProductIcon` here. Since the products are not yet live, there's no way a user would get this error even right now. But yeah - if #37760 is merged first, it'd be the best.